tests/rptest/tests/node_status_test.py

```diff
@@ -0,0 +1,175 @@
+# Copyright 2020 Redpanda Data, Inc.
```

Comment:
>tests: add node status ducktape tests

can we sequence this commit after all of the feature which it depends on is added?

Reply:
Yeah. I don't know why the test commit ended up there. Must have made a mistake when rebasing.